.claude/prompts/tune-constants.md

```
@@ -0,0 +1,60 @@
+# Prompt: Update Tuning Constants
+
+Use this prompt when you have measured new values and want to update constants safely.
+Fill in all bracketed values. Do not use this prompt to guess at values.
+
+---
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)